src/Dommel/SqlExpression.cs

```diff
@@ -419,6 +421,22 @@ protected virtual object VisitContainsExpression(MethodCallExpression expression
             return $"lower({column}) like lower({paramName})";
         }
 
+        /// <summary>
+        /// Processes ToString expression to CAST columns into Char before comparison.
+        /// </summary>
+        /// <param name="expression"></param>
+        /// <returns></returns>
+        protected virtual object VisitToStringExpression(MethodCallExpression expression)
+        {
+            var column = VisitExpression(expression.Object);
+            if (expression.Arguments.Count >= 1)
+            {
+                throw new ArgumentException("ToString-expression should not contain any argument.", nameof(expression));
```

Comment:
Can you add a test case which verifies this behavior? E.g.
```cs
var ex = Assert.Throws<ArgumentException>(() => _sqlExpression.Where(p => p.CategoryId.ToString("n2") == "1"));
Assert.Equal("ToString-expression should not contain any argument.", ex.Message);
```

Reply:
done.